### ISSUE TITLE
After recent commit, type compilation error

### ISSUE THREAD

**Issue by don-p** [diogofcunha/react-virtualized-tree#69]
```
react-virtualized-tree/index.d.ts
(36,21): Generic type 'RendererProps<T>' requires 1 type argument(s).
```

This is caused by incorrect change in commit f6fe1a429c821015963c49b40ddb779d370d888d
https://github.com/diogofcunha/react-virtualized-tree/pull/68/commits/f6fe1a429c821015963c49b40ddb779d370d888d
```
- type onChange = (updateParams: NodeAction) => void;
+ type onChange = (nodes: Node[], node: Node) => Node[];

export interface RendererProps<T> {
  - onChange: onChange;
  + onChange: (updateParams: NodeAction) => void;
```

onChange param signature is still not matching between the type definition and the property, and on line # 36 this:

`  children: (props: RendererProps) => JSX.Element;` 
should be something like this:
`  children: (props: RendererProps<any>) => JSX.Element;`

**Comment by diogofcunha** [comment]
hmm, what's you TS version, on 3 it generally defaults T generally defaults to {}

**Comment by diogofcunha** [comment]
Thanks for reporting, fixed on `2.3.7` 😃
